### PR TITLE
docs(plugins): update internal plugins

### PR DESCRIPTION
**Pull request by chenxsan**
I've updated some outdated info. Also added some inline comments for easier review.

### CODE REVIEW

File: src/content/plugins/internal-plugins.md [hunk 1]
```diff
@@ -34,13 +35,11 @@ Applies Node.js style filesystem to the compiler.
 
 Plugins affecting the compiler
 
-### CachePlugin
```
Comment (chenxsan) [inline src/content/plugins/internal-plugins.md:37]: Related commit here https://github.com/webpack/webpack/pull/8100/commits/7340fbb5476f9b35432020a20e8059b2b86e854a

File: src/content/plugins/internal-plugins.md [hunk 2]
```diff
@@ -64,12 +63,6 @@ Plugins, which add entry chunks to the compilation.
 
 Adds an entry chunk on compilation. The chunk is named `chunkName` and contains only one module (plus dependencies). The module is resolved from `request` in `context` (absolute path).
 
-### MultiEntryPlugin
```
Comment (chenxsan) [inline src/content/plugins/internal-plugins.md:67]: It's removed in this commit https://github.com/webpack/webpack/commit/6d5310fbd609f60ce36dc3611dbe9a76f61a0848

File: src/content/plugins/internal-plugins.md [hunk 3]
```diff
@@ -78,16 +71,6 @@ Prefetches `request` and dependencies to enable a more parallel compilation. It
 
 ## output
 
-### FunctionModulePlugin
```
Comment (chenxsan) [inline src/content/plugins/internal-plugins.md:81]: Removed in https://github.com/webpack/webpack/commit/69a545c4444d881c87e6462ea6da707f7793fc93

File: src/content/plugins/internal-plugins.md [hunk 4]
```diff
@@ -168,10 +151,6 @@ Make webpack_public_path, webpack_require, webpack_modules and webpack_chunk_loa
 
 Currently useless. Ensures compatibility with other module loaders.
 
-### ConsolePlugin
```
Comment (chenxsan) [inline src/content/plugins/internal-plugins.md:171]: It's moved into `NodeSourcePlugin` in https://github.com/webpack/webpack/commit/c4140aac51ace25944a74d0cbad00ca7cf5540f4

File: src/content/plugins/internal-plugins.md [hunk 5]
```diff
@@ -245,12 +224,6 @@ Provides AMD-style `define` and `require` to modules. Also bind `require.amd`, `
 
 Provides CommonJs-style `require` to modules.
 
-### LabeledModulesPlugin
```
Comment (chenxsan) [inline src/content/plugins/internal-plugins.md:248]: Removed in https://github.com/webpack/webpack/commit/a14e563f67292cf2bdfe28b01abd8cb5fee1e20e

File: src/content/plugins/internal-plugins.md [hunk 6]
```diff
@@ -313,14 +286,6 @@ Merges chunks until each chunk has the minimum size of `minChunkSize`.
 
 Adds chunk ids of chunks which are included in the chunk. This eliminates unnecessary chunk loads.
 
-### OccurenceOrderPlugin
```
Comment (chenxsan) [inline src/content/plugins/internal-plugins.md:316]: Removed in https://github.com/webpack/webpack/commit/a2af69056ae3a477f65f9d3dbdf327e2d3c6e215

File: src/content/plugins/internal-plugins.md [hunk 7]
```diff
@@ -312,17 +285,3 @@ Merges chunks until each chunk has the minimum size of `minChunkSize`.
...
-
-### OccurenceOrderPlugin
-
-`optimize/OccurrenceOrderPlugin(preferEntry)`
-
-Order the modules and chunks by occurrence. This saves space, because often referenced modules and chunks get smaller ids.
-
-`preferEntry` If true, references in entry chunks have higher priority
-
-### DedupePlugin
```
Comment (chenxsan) [inline src/content/plugins/internal-plugins.md:324]: Removed in https://github.com/webpack/webpack/commit/67f31aeebc8b90d48e6733bea60d2203c1b8507d

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**